web/app/views/bloggs/index.html.erb

```diff
@@ -2,6 +2,21 @@
 
 <%= link_to 'New article', new_blogg_path %>
 
+
+<% if flash[:error] %>
+    <p class="notice"><%= flash[:error] %></p>
+<% end %>
+
+<% if user_signed_in? %>
+    <li>
+      <%= link_to('Logout', destroy_user_session_path, :method => :delete) %>
+    </li>
+<% else %>
+    <li>
+      <%= link_to('Login', new_user_session_path)  %>
```

Comment:
2 spaces @MattyAyOh? 2 spaces? Really? You're killing me smalls! :stuck_out_tongue_closed_eyes: